app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -200,70 +206,69 @@ fun CreateActivityScreen(
+                      .align(Alignment.CenterHorizontally),
           ) {
-            Button(
-                onClick = { showDialog = true },
-                modifier = Modifier.width(300.dp).height(40.dp).testTag("addAttendeeButton"),
+            Row(
+                horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
+                verticalAlignment = Alignment.CenterVertically,
             ) {
-              Row(
-                  horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
-                  verticalAlignment = Alignment.CenterVertically,
-              ) {
-                Icon(
-                    Icons.Filled.Add,
-                    contentDescription = "add a new attendee",
-                )
-                Text("Add Attendee")
-              }
+              Icon(
+                  Icons.Filled.Add,
+                  contentDescription = "add a new attendee",
+              )
+              Text("Add Attendee")
             }
-            if (attendees.isNotEmpty()) {
-              LazyRow(
-                  modifier = Modifier.fillMaxHeight().height(85.dp).padding(8.dp),
-              ) {
-                items(attendees.size) { index ->
-                  Row(
-                      modifier =
-                          Modifier.padding(8.dp)
-                              .background(Color(0xFFFFFFFF))
-                              .testTag("attendeeRow${index}"),
-                  ) {
-                    Text(
-                        text = attendees[index].name,
-                        modifier = Modifier.padding(8.dp).testTag("attendeeName${index}"),
-                        style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
-                    )
-                    Spacer(modifier = Modifier.width(8.dp))
-                    Text(
-                        text = attendees[index].surname,
-                        modifier = Modifier.padding(8.dp).testTag("attendeeSurname${index}"),
-                        style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
-                    )
-                    Spacer(modifier = Modifier.width(8.dp))
-                    Text(
-                        text = attendees[index].age.toString(),
-                        modifier = Modifier.padding(8.dp).testTag("attendeeAge${index}"),
-                        style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 12.sp),
-                    )
-                    Button(
+          }
+          if (attendees.isNotEmpty()) {
+            LazyRow(
+                modifier = Modifier.fillMaxHeight().height(85.dp).padding(8.dp),
+            ) {
+              items(attendees.size) { index ->
+                Card(
+                    modifier =
+                        Modifier.padding(8.dp)
+                            .background(Color(0xFFFFFFFF))
+                            .testTag("attendeeRow${index}"),
```

Comment:
Switching from Row to Card for displaying attendee information is a significant improvement. It makes each attendee entry visually distinct and easier to interact with, which improves the UI's aesthetic and functional aspects.